sphericart/CMakeLists.txt

```diff
@@ -30,13 +32,44 @@ if (${CMAKE_CURRENT_SOURCE_DIR} STREQUAL ${CMAKE_SOURCE_DIR})
     endif()
 endif()
 
-add_library(sphericart
+set(COMMON_SOURCES
```

Comment:
Instead of adding to the sources, I think you can keep the library declaration and add to it with `target_sources`, like we do for sphericart-torch.

Reply:
still valid